components/frontend/src/source/SourceEntities.test.js

```diff
@@ -105,6 +105,15 @@ it('sorts the entities by status rationale', async () => {
     assertOrder(["A", "B", "C"])
 })
 
+it('sorts the entities by first seen date', async () => {
+    render(<DataModel.Provider value={data_model}><SourceEntities metric={metric} source={source} /></DataModel.Provider>)
+    assertOrder(["C", "B", "A"])
+    await userEvent.click(screen.getByText(/first seen/))
```

Comment:
But from line 40 it seems that none of these entities have "first seen" attributes. If first seen would be the same as the datetime, I'd expect the order to be C-A-B.